src/json/members/nikhil.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Nikhil Ruia",
+    "pennkey": "nruia",
+    "major": "Computer Science",
+    "school": "SEAS",
+    "bio": "Hi! My hobbies include cars, movies, and karaoke with friends. I recently started cooking too, but that's not going as well. I am also involved in UPMUNC.",
+    "hometown": "Glenview, IL",
+    "team": "Penn Courses",
+    "roles": ["Backend Developer"],
+    "photo": "https://imgur.com/a/aMYgh9l",
```

Comment:
looks like this photo is the problem, you'll need to use the link to the actual image file (should end with .jpg), not the link to the imgur page